pkg/commands/patch/patch_parser.go

```diff
@@ -98,35 +98,35 @@ func (l *PatchLine) render(selected bool, included bool) string {
 		return coloredString(style.FgCyan, match[1], selected, included) + coloredString(theme.DefaultTextColor, match[2], selected, false)
 	}
 
-	colorAttr := theme.DefaultTextColor
+	textStyle := theme.DefaultTextColor
```

Comment:
I'm calling these things textStyles now so that the variable name is more accurate